.docker/Dockerfile-template.ckan

```diff
@@ -0,0 +1,32 @@
+FROM openknowledge/ckan-dev:@CKAN_VERSION@
```

Comment:
how does this file work when running locally?

Reply:
You're meant to run it via `.circleci/build.sh`, which performs the substitution.

I suppose you could manually create `.docker/Dockerfile.ckan` instead, if you really wanted to.

Reply:
update readme on how you run this locally please.